### ISSUE TITLE
Statsd input TCP listener

### ISSUE THREAD

**Issue by szibis** [influxdata/telegraf#2208]
## Feature Request

For more reliable communication it would be very nice to have TCP listener in statsd input. 

### Proposal:

Adding TCP listener inside statsd input plugin

### Use case: [Why is this important (helps with prioritizing requests)]

This is especially good idea when we use Telegraf statsd input as centralised statsd server with multiple outputs on the end and Telegraf will get data from multiple places. 

I can try to implement this but asking before any work will be done.

**Comment by sparrc** [comment]
PRs welcome :)

**Comment by szibis** [comment]
https://github.com/influxdata/telegraf/pull/2293
